### PR TITLE
Upgrade to Django4

### PR DESCRIPTION
**Pull request by gracieflores**
One change in the `requirements.txt` file is temporary, and that is the installing of `django-premis-event-service` django4 branch. One other change that I wasn't sure about was the `wsgi.py` file (`SITE_PACKAGES` change).

@ldko @somexpert This is ready for review.

### CODE REVIEW

**Review by ldko — APPROVED** [review]
I ran this in Docker, POSTed a couple bags and queue entry. It works for me.

**Review by somexpert — APPROVED** [review]
Changes look good, including the wsgi changes, and the tests ran for me in podman. :thumbsup:

**Comment by gracieflores** [comment]
@ldko @somexpert Since I have made changes to the settings, would you please review those changes. The changes were successful when I ran the podman instance and the tests passed.

**Comment by ldko** [comment]
@gracieflores do the changes to the settings still have the Django Debug Toolbar showing on the local deployment?

File: coda/config/settings.py [hunk 1]
```diff
@@ -47,7 +20,13 @@ def get_secret(setting, secrets=secrets):
         raise ImproperlyConfigured(error_msg)
 
 
-DEBUG = True
+DEBUG = get_secret('SECRET_KEY')
```
Comment (ldko) [inline coda/config/settings.py:23]:
```suggestion
DEBUG = get_secret('DEBUG')
```

**Comment by ldko** [comment]
We should also update secrets.json.template if new secrets are required.

**Comment by somexpert** [comment]
Changes look good to me, and the app and tests ran fine locally :thumbsup:

**Review by ldko — APPROVED** [review]
Thanks, Gracie! :+1:

**Pull request merged**